### ISSUE TITLE
Only 200 status code in account.connect

### ISSUE THREAD

**Issue by noglik** [Tezirg/os2#1]
Hello
I noticed, that in connect function, only 200 response code is allowed. 
```javascript
if (response.statusCode !== 200) {
  reject(new Error(response.statusMessage));
  return;
}
```
CDN, that i'm used returns 204 on connect.
Can you expand status code range on all 200? If it's OK, i can make pull request.
Thank you.

**Comment by Tezirg** [comment]
Hello,

Nice catch! It seems that 204 is returned when the account doesnt have any containers.
https://library.netapp.com/ecmdocs/ECMP12481627/html/GUID-6F45BD29-F01C-4B1D-AA99-3F37E92542C5.html

I'll accept your pull request fixing the account connect statusCode. 
It should be the only place where it happens, but feel free to open a new issue if that happens somewhere else.

**Comment by noglik** [comment]
What a great commit :smile: .
Btw, i noticed that you have broken docker containers. Maybe i could fix it soon.

**Comment by Tezirg** [comment]
Thank you for your contribution, I have published the fix in 0.0.12, now available on the npm package manager.
Yes I'm still working on the docker environement because I don't have access to a swift server anymore. 
The goal is to automate the tests with circle.ci. 
I'll be happy to accept a working pull request
